target/start-mailserver.sh

```diff
@@ -944,31 +944,39 @@ function _setup_dkim() {
 function _setup_ssl() {
 	_notify 'task' 'Setting up SSL'
 
+  function apply_tls_level() {
```

Comment:
Sounds nip-picky, but please use the `{` on the next line for consistency

Reply:
Sorry, I don't quite understand what you're asking here? Do you want this:

```sh
function _setup_ssl() {
    _notify 'task' 'Setting up SSL'

    function apply_tls_level()
    {
```

I am not sure how that would be consistent, but I'm not sure how else to interpret the request for `{` to be moved to the next line?

Reply:
I just came back because I figured this post would be counter-intuitive.

``` BASH
function _setup_ssl() {
    _notify 'task' 'Setting up SSL'

    function apply_tls_level()
    {
```

is correct. Have a brief look at the current `start-mailserver.sh` script. I refactored it in #1605, and introduced the styling guide and linting. Because this PR is older, `function _setup_ssl() {` is still shown in the diff, but by now, it's

``` BASH
function _setup_ssl()
{
```

in the actual script.

Reply:
Oh, I see, new changes since my recent rebase. Not a fan of that style personally, but no worries I'll rebase and match the new style :+1:

Reply:
Thank you very much, I know, opinions differ on this, but nevertheless, I'm happy you can adjust it! This is therefore resolved.